packages/common/lib/types/v1_0_09.types.ts

```diff
@@ -15,19 +16,10 @@ export interface AuthorizationRequestV1_0_09 extends CommonAuthorizationRequest
   op_state?: string;
 }
 
-export interface AuthorizationDetailsJwtVcJsonV1_0_09 extends CommonAuthorizationDetails {
-  // If the Credential Issuer metadata contains an authorization_server parameter, the authorization detail's locations common data field MUST be set to the Credential Issuer Identifier value.
-  locations?: string[];
-  types: string[];
-  // fixme: we don't support this property in the current flow for jff. so I commented it out
-  //CredentialSubject?: IssuerCredentialSubject;
-  [key: string]: unknown;
-}
-
 export function isAuthorizationRequestV1_0_09(request: CommonAuthorizationRequest): boolean {
   return request && 'op_state' in request;
 }
 
-export function isCredentialOfferV1_0_09(request: CredentialOfferV1_0_09 | CredentialOfferV1_0_11): boolean {
-  return request && 'issuer' in request && 'op_state' in request;
+export function isCredentialOfferV1_0_09(request: CredentialOfferPayload): boolean {
```

Comment:
Probably wise to create an isCredentialOfferVersion(request, version) method as a TODO, for these, otherwise we end up with 10 of them in the future.

Also having something like isCredentialOfferAtLeast or something makes sense for the future

Reply:
added two tickets for `CredentialOffer` and `AuthorizationRequest`